### PR TITLE
[DependencyInjection] Removed someone elseif keywords

### PR DESCRIPTION
**Pull request by sergiors**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: src/Symfony/Component/DependencyInjection/ContainerBuilder.php [hunk 1]
```diff
@@ -1062,13 +1074,12 @@ private function shareService(Definition $definition, $service, $id)
 
     private function getExpressionLanguage()
     {
-        if (null === $this->expressionLanguage) {
-            if (!class_exists('Symfony\Component\ExpressionLanguage\ExpressionLanguage')) {
-                throw new RuntimeException('Unable to use expressions as the Symfony ExpressionLanguage component is not installed.');
-            }
-            $this->expressionLanguage = new ExpressionLanguage(null, $this->expressionLanguageProviders);
+        if (!class_exists('Symfony\Component\ExpressionLanguage\ExpressionLanguage')) {
```
Comment (sstok) [inline src/Symfony/Component/DependencyInjection/ContainerBuilder.php]: This will require a function call for every time this method is called.

Comment (sergiors) [inline src/Symfony/Component/DependencyInjection/ContainerBuilder.php]: Make sense.

File: .idea/vcs.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (jakzal) [inline .idea/vcs.xml]: This file should be removed.

Comment (sergiors) [inline .idea/vcs.xml]: Oh sorry

**Comment by fabpot** [comment]
:-1: It does not bring any value and makes merging other branches much more difficult.

**Comment by sergiors** [comment]
@fabpot thanks for your feedback :)

**Pull request closed without merging**